config/v1/types_infrastructure.go

```diff
@@ -478,6 +478,20 @@ type AWSPlatformStatus struct {
 	// +kubebuilder:validation:MaxItems=25
 	// +optional
 	ResourceTags []AWSResourceTag `json:"resourceTags,omitempty"`
+
+	// userDNS indicates if the cluster is using a DNS solution configured
+	// by the user in place of Route53.
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	// +optional
+	UserDNS string `json:"userDNS,omitempty"`
+
+	// apiLBName contains the name of the API LB
+	// +optional
+	ApiLBName string `json:"apiLBName,omitempty"`
+
+	// apiIntLBName contains the name of the API-Int LB
+	// +optional
+	ApiIntLBName string `json:"apiIntLBName,omitempty"`
```

Comment:
I can see us wanted to add more load balancer options in the future, why not create something structured
```
apiLoadBalancers:
  internal:
    name: blah
  external:
    name: blah
```
This would be more future proof and allow us to add additional configuration in the future